src/HTMLParser.php

```diff
@@ -510,6 +522,37 @@ public function getMainImage()
         return false;
     }
 
+    /**
+     * @return array
+     */
+    public function getImages()
+    {
+        $result = [];
+        if (!empty($this->metadata['image'])) {
+            $result[] = $this->metadata['image'];
+        }
+        if (null == $this->dom) {
+            return $result;
+        }
+
+        foreach ($this->dom->getElementsByTagName('img') as $img) {
+            if ($src = $img->getAttribute('src')) {
+                $result[] = $src;
+            }
+        }
+
+        if ($this->getConfig()->getOption('fixRelativeURLs')) {
+            list($pathBase, $scheme, $prePath) = $this->getPathInfo($this->getConfig()->getOption('originalURL'));
```

Comment:
This seems to be dead code. Those three vars are not used later in this function.

Reply:
Thanks, fixed on commit
https://github.com/andreskrey/readability.php/pull/30/commits/4f0b60a3b39995d822359ea6e4fefa4b5751154e